test/specs/map/Layers.spec.js

```diff
@@ -845,10 +866,12 @@ describe('ga_layers_service', function() {
           var source = layer.getSource();
           expect(source instanceof ol.source.Vector).to.be.ok();
           expectCommonProperties(layer, 'geojson');
-          gaLayers.getLayerPromise('geojson').then(function(feats) {
-            expect(feats[0].getGeometry().getCoordinates()).to.eql([2600000, 1199999.999848965]);
+          $q.all([gaLayers.getLayerPromise('geojson'),
+            gaLayers.getLayerStylePromise('geojson')]).then(function(vals) {
+            expect(vals[0][0].getGeometry().getCoordinates()).to.eql([2600000, 1199999.999848965]);
```

Comment:
maybe you could test the layer's style used too. Be sure it's the one we just load.

Reply:
test added